mkdocs.yml

```
@@ -175,3 +177,11 @@ extra_css:
 - assets\stylesheets\links.css
 extra_javascript:
 - assets\javascripts\version-select.js
+plugins:
+  - search
+  ## uncomment the next lines to run the HTML Proofer in development to check for bad links
+  # - htmlproofer:
+  #     raise_error_excludes:
+  #       500: ['*']
+  #       404: ['https://support.google.com/accounts/answer/185833?hl=en']
+  #       404: ['https://support.google.com/accounts/answer/6010255?hl=en']
```

I get a false negative for a 3rd URL:
```suggestion
  #       404:
  #         - 'https://support.google.com/accounts/answer/185833?hl=en'
  #         - 'https://support.google.com/accounts/answer/6010255?hl=en'
  #         - 'https://support.google.com/accounts/answer/185833?hl=en'
```